src/nvim/ex_cmds.c

```diff
@@ -5046,6 +5008,46 @@ int find_help_tags(char_u *arg, int *num_matches, char_u ***matches, int keep_la
   return OK;
 }
 
+// Called when starting to edit a buffer for a help file.
+static void prepare_help_buffer(void)
```

Comment:
Ok, I've seen #710, and apparently we do put Doxygen comments in the implementation and generate the headers. I'll change it to `///` now.